api/scripts/tcr/tcr.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+alteredFilePaths=$(git ls-files -m '*.js');
+files=$@
```

Comment:
`$@` peut contenir plusieurs éléments, une variable en contient un seul. Note : nécessite un changement plus bas.
```suggestion
files=( "$@" )
```
Plus d'infos :
- https://github.com/koalaman/shellcheck/wiki/SC2124

Reply:
Done.